src/main/java/org/apache/pulsar/manager/zuul/HttpsClientConfiguration.java

```diff
@@ -13,10 +13,16 @@
  */
 package org.apache.pulsar.manager.zuul;
 
+import org.apache.http.client.methods.HttpDelete;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpHead;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.client.methods.HttpPut;
```

Comment:
Remove unused imports.

Reply:
@nodece Thanks for pointing it out. Removed the unused imports.